README.md

```diff
@@ -6,14 +6,19 @@ Copy `OnBaseDocsApi/api-config-example.yaml` to `OnBaseDocsApi/api-config.yaml`
 
 ## Run the API
 
-### Docker
+Example Dockerfile:
```

Comment:
Why not have a seperate `example_dockerfile` like we do with configs?

Reply:
I thought about that too, but I think it implies we have to copy `example_dockerfile` to `Dockerfile` and place under this repo to run this API. Ideally, we should use only specify necessary commands in this section, but the steps are very different between systems, so I guess using docker is still the most ideal way for this section for now.